vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -359,15 +348,6 @@ func TestStandardTxExecutorAddDelegator(t *testing.T) {
 			}
 			err = tx.Unsigned.Visit(&executor)
 			require.ErrorIs(err, tt.expectedExecutionErr)
-
-			mempoolExecutor := MempoolTxVerifier{
```

Comment:
We run all the txs through the `StandardTxExecutor` already which the `MempoolTxVerifier` called internally anyway. This can be removed outside of this PR.